modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -9,47 +9,46 @@ A message type in Partner Manager can be used in one of the following ways:
 * Source at host
 * Send to partners
 
-Message types are always managed within the host partner profile page, regardless of whether they are used specifically for a single partner or across multiple trading partners. Message types are reusable across message flows for different trading partners.
+Message types are always managed in the host profile page, regardless of whether they are used for a single partner or for multiple trading partners. Additionally, message types are reusable across message flows for different trading partners.
 
-Create message types from the Host page or from the message flow configuration wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
+You can create message types from the Host page or from the Message Flow wizard. When you create message types during message flow configuration, Partner Manager automatically designates the usage for the message type.
```

Comment:
GLOBALLY reconcile/distinguish among the terms host, host page, Host page, host profile page....

Reply:
Ditto comments about automatic designation vs user configuration; GLOBAL
GLOBALLY clarify whether "usage" actually means specifying incoming vs outgoing, or message "type".

Reply:
Done